packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -150,7 +151,10 @@ export default {
       isPersonalLocation: useActiveLocation(isLocationSpacesActive, 'files-spaces-personal-home'),
       isPublicLocation: useActiveLocation(isLocationPublicActive, 'files-public-files'),
       isSpacesProjectsLocation: useActiveLocation(isLocationSpacesActive, 'files-spaces-projects'),
-      isSpacesProjectLocation: useActiveLocation(isLocationSpacesActive, 'files-spaces-project')
+      isSpacesProjectLocation: useActiveLocation(isLocationSpacesActive, 'files-spaces-project'),
+      ...useAppDefaults({
```

Comment:
@kulmann this was my "fix" for having `makeRequest` in here and the reason I said it was "quick" and that you will "have comments" on it.
I tried separating things to not need to apply the full "defaults", but it was becoming messy, I don't think I managed to make it work and didn't have more time to spend on it. So if you have suggestions, they are welcomed. (if this is ok for you, even better).

Reply:
The refactoring into `makeRequest` feels like a huge scope creep for this PR and makes it much harder to review. Also we have some different ideas about making network requests. Will allow it for this PR but will refactor it soon...

Reply:
The refactoring into `makeRequest` feels like a huge scope creep for this PR and makes it much harder to review. Also we have some different ideas about making network requests. Will allow it for this PR but will refactor it soon...